pkg/iac/scanners/cloudformation/parser/reference.go

```diff
@@ -19,40 +16,6 @@ func NewCFReference(id string, resourceRange iacTypes.Range) CFReference {
 	}
 }
 
-func NewCFReferenceWithValue(resourceRange iacTypes.Range, resolvedValue Property, logicalId string) CFReference {
-	return CFReference{
-		resourceRange: resourceRange,
-		resolvedValue: resolvedValue,
-		logicalId:     logicalId,
-	}
-}
-
 func (cf CFReference) String() string {
 	return cf.resourceRange.String()
```

Comment:
Were they not used?

Reply:
The `resolvedValue` field was used in methods that were not used anywhere else.